<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemEditDialog</class>
 <widget class="QDialog" name="itemEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑</string>
  </property>
  <widget class="QLineEdit" name="pltEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>平台:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>密码中允许出现的字符</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>自定义:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="symbolChk">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>特殊字符</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="upperChk">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>大写字母</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="allowedEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="numberChk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>数字</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lowerChk">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>小写字母</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码的最大长度:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="md5lenEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>220</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linkEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>链接:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
